packages/dapp/src/components/Navbar/Menu/Menus.tsx

```diff
@@ -1,25 +1,57 @@
+import { Snackbar } from '@mui/material';
+import MuiAlert from '@mui/material/Alert';
 import { useMenuStore } from '@transferto/dapp/src/stores/menu';
 import { SyntheticEvent } from 'react';
 import { ChainsMenu, MainMenu, WalletMenu, WalletSelectMenu } from '.';
 import { SupportModal } from '../../SupportModal';
 
 export const Menus = () => {
-  const anchorRef = useMenuStore((state) => state.anchorRef);
-
+  const [anchorRef, openSnackbar, onOpenSnackbar] = useMenuStore((state) => [
+    state.anchorRef,
+    state.openSnackbar,
+    state.onOpenSnackbar,
+  ]);
   const handleClose = (event: Event | SyntheticEvent) => {
     event.preventDefault();
     if (anchorRef.contains(event.target as HTMLElement)) {
       return;
     }
   };
 
+  const handleCloseSnackbar = (
+    event?: React.SyntheticEvent | Event,
+    reason?: string,
+  ) => {
+    if (reason === 'clickaway') {
+      return;
+    }
+    onOpenSnackbar(false);
+  };
+
+  console.log(openSnackbar);
+
   return (
     <>
       <MainMenu handleClose={handleClose} />
       <WalletMenu handleClose={handleClose} />
       <WalletSelectMenu handleClose={handleClose} />
       <ChainsMenu handleClose={handleClose} />
       <SupportModal />
+      <Snackbar
+        open={openSnackbar.open}
+        autoHideDuration={2000}
+        onClose={handleCloseSnackbar}
+        anchorOrigin={{ vertical: 'top', horizontal: 'right' }}
+        sx={{ top: '80px !important' }}
```

Comment:
Why do we need `!important` here?

Reply:
We don´t it just had been like that ever since I started to use MUI Snackbar. I have now adjusted it in the theme itself with styleOverrides of the Snackbar component.